### PR TITLE
Completes GET /api/articles/:article_id/comments Happy & Sad paths & …

### PR DESCRIPTION
**Pull request by danrfletcher**
…refactored tests for better organization.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -8,62 +8,110 @@ beforeAll(() => seed(testData));
...
+            .then((res) => {
+                expect(res.body.comments.length).toBeGreaterThanOrEqual(11);
+                expect(res.body.comments).toBeSorted('created_at', 'desc')
+                res.body.comments.forEach((comment) => {
+                    expect(typeof comment.comment_id).toBe('number');
+                    expect(typeof comment.article_id).toBe('number');
+                    expect(typeof comment.body).toBe('string');
+                    expect(typeof comment.votes).toBe('number');
+                    expect(typeof comment.author).toBe('string');
+                    expect(typeof comment.created_at).toBe('string');
```
Comment (aloush-dev) [inline __tests__/app.test.js]: This works and its fine, but if it hasn't been mentioned already I'd start looking into expect objectcontaining and tomatchobject matchers for jest they allow for much more robust testing.

**Review by aloush-dev — APPROVED** [review]
Awesome job.
Merge when you're ready.

**Review by aloush-dev — APPROVED** [review]
Awesome job.
Merge when you're ready.

**Comment by danrfletcher** [comment]
@itsaliabdallah thanks for the feedback, just implemented objectContaining & merged!

**Pull request merged**